components/collator/src/elements.rs

```diff
@@ -716,6 +716,11 @@ impl CharacterAndClassAndTrieValue {
 /// Note that 0xFF is won't be assigned to an actual
 /// canonical combining class per definition D104
 /// in The Unicode Standard.
+//
+// NOTE: The Pernosco debugger has special knowledge
```

Comment:
nit: this comment should also go on the `mod elements` line in the crate root

Reply:
Opened #2619 for this.